client/components/navigation/Navbar.tsx

```diff
@@ -0,0 +1,187 @@
+import NextLink from 'next/link';
```

Comment:
this file is duplicated and in the wrong folder/location

Reply:
Where is the duplicate to this?

Reply:
The file, not the line haha